editor.planx.uk/src/routes/views/draft.tsx

```diff
@@ -69,7 +69,7 @@ const fetchSettingsForDraftView = async (
                 favicon
               }
               name
-              teamSettings: team_settings {
+              settings: team_settings {
```

Comment:
I'd expect these changes to also need to be reflected in the store? Take a look at where `team.teamSettings` is being referred to (`setTeam()` I think) 👍

Reply:
Ah @RODO94 sorry - just realised this is typed as `Team.teamSettings` in planx-core and I missed it there when reviewing. Please ignore https://github.com/theopensystemslab/planx-new/pull/3334#discussion_r1661227147 for now in the name of expediency and we can take another look at this later.

Reply:
Yeah, maybe for another PR?